assets/images/team26/2023-01-29-team26-deepfake-gen.md.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+comments: true
+title: Post Template
+author: UCLAdeepvision
+date: 2022-01-18
+---
+
```

Comment:
Your post format is incorrect. Please double-check it with other students' posts.

And please do not delete the line of **<!--more-->** from the md file.